packages/@aws-cdk/cx-api/FEATURE_FLAGS.md

```diff
@@ -118,7 +119,8 @@ The following json shows the current recommended set of flags, as `cdk init` wou
     "@aws-cdk/aws-appsync:useArnForSourceApiAssociationIdentifier": true,
     "@aws-cdk/aws-rds:preventRenderingDeprecatedCredentials": true,
     "@aws-cdk/aws-codepipeline-actions:useNewDefaultBranchForCodeCommitSource": true,
-    "@aws-cdk/aws-cloudwatch-actions:changeLambdaPermissionLogicalIdForLambdaAction": true
+    "@aws-cdk/aws-cloudwatch-actions:changeLambdaPermissionLogicalIdForLambdaAction": true,
+    "@aws-cdk/aws-codepipeline:crossAccountKeysDefaultValueToFalse": true
```

Comment:
```suggestion
    "@aws-cdk/aws-codepipeline:crossAccountKeysDefaultValueToFalse": true,
```

Reply:
This is in json format, not javascript. So putting the trailing comma in the last line violates the json spec. I did not reflect this as I have seen [other lines](https://github.com/go-to-k/aws-cdk/blob/5f804eec17547e97993e028064c65cc97cb5dc3e/packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md#currently-recommended-cdkjson) do the same. (CodeBuild CI doesn't error...) What do you think?

```
```json
{
  "context": {
    "@aws-cdk/aws-lambda:recognizeLayerVersion": true,
    "@aws-cdk/core:checkSecretUsage": true,
    "@aws-cdk/core:target-partitions": [
      "aws",
      "aws-cn" <-- here
    ],
```

> JSON, however, disallows all trailing commas.

> As JSON is based on a very restricted subset of JavaScript syntax, trailing commas are not allowed in JSON.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
I actually changed cdk.json and ran it and got an error.
```
cdk.json: Unexpected token } in JSON at position 2049
error Command failed with exit code 1.
```

Reply:
Ah yes you're right here, this is fine then